mindsdb/integrations/handlers/sentence_transformer_handler/helpers.py

```diff
@@ -0,0 +1,10 @@
+from integrations.handlers.sentence_transformer_handler.settings import (
+    load_embeddings_model,
```

Comment:
Nit: maybe `load_embeddings_model` should be moved here, instead?

Reply:
I like to follow pattern where any loading and config happens in `settings.py`, ofc this is not set in stone so happy to move it if you prefer it that way

Reply:
Ok fair, let's do that, no strong preference on my end 👌